File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        top = []
        my_dict = defaultdict(int)
        for num in nums:
            my_dict[num] += 1
        
        for i in range(k):
            max_val_key = max(my_dict, key=my_dict.get)
            top.append(max_val_key)
            my_dict.pop(max_val_key)
        
        return top

        

File: Data Structures & Algorithms/maximum-ascending-subarray-sum/submission-0.java
class Solution {
    public int maxAscendingSum(int[] nums) {

        if (nums.length == 0) {
            return 0;
        }
        int max = 0;
        int cur = nums[0];
        for (int right = 1; right < nums.length; right++) {
            if (nums[right] > nums[right - 1]) {
                cur += nums[right];
            } else {
                max = Math.max(max, cur);
                cur = nums[right];
            }
        }

        return Math.max(max, cur);
        
    }
}